Update 01-login.md (#9405)
auth0/docs: articles/quickstart/spa/react/01-login.md

import { useAuth0 } from "@auth0/auth0-react";

const Profile = () => {
  const { user, isAuthenticated, isLoading } = useAuth0();

>   if(isLoading){

Reviewer: Whitespace

```suggestion
  if (isLoading) {
```

export default Profile;
```

> The `user` property contains sensitive information and artifacts related to the user's identity. As such, its availability depends on the user's authentication status. To prevent any render errors, use the `isAuthenticated` property from `useAuth0()` to check if Auth0 has authenticated the user before React renders any component that consumes the `user` property. Make sure that the Auth0 SDK has loaded before trying to use isAuthenticated. It will return false if the SDK has not loaded yet. You can use the isLoading property to detect that and handle it. 

Reviewer: ```suggestion
The `user` property contains sensitive information and artifacts related to the user's identity. As such, its availability depends on the user's authentication status. To prevent any render errors, use the `isAuthenticated` property from `useAuth0()` to check if Auth0 has authenticated the user before React renders any component that consumes the `user` property. Ensure that the SDK has completed loading before accessing the `isAuthenticated` property, by checking that `isLoading` is `false`.
```
